docs/en/04_Changelogs/4.0.0.md

```diff
@@ -63,6 +63,7 @@
  * Search filter classes (e.g. `ExactMatchFilter`) are now registered with `Injector`
    via a new `DataListFilter.` prefix convention.
    see [search filter documentation](/developer_guides/model/searchfilters) for more information.
+ * FormField templates no longer look in the 'forms' folder for templates.
```

Comment:
Should mention the changed semantics of `SSViewer:: get_templates_by_class()` (returning all templates regardless of existence)